lib/cocoapods/generator/embed_frameworks_script.rb

```diff
@@ -153,6 +153,14 @@ def script
               fi
             fi
           }
+          
+          # Copies the bcsymbolmap files of a vendored framework
+          install_bcsymbolmap() {
+              local bcsymbolmap_path = "$1"
+              local destination="${TARGET_BUILD_DIR}"
```

Comment:
is this the right destination? `.frameworks` go into `"${TARGET_BUILD_DIR}/${FRAMEWORKS_FOLDER_PATH}"`. Where does Xcode place `bcsymbolmap` files without CocoaPods for example?

Reply:
@dacaiguoguogmail what is the answer here?

Reply:
@dnkoutso I test the result is right. The destination is `${TARGET_BUILD_DIR}`. I will do more tests.

Reply:
See my comment below with screenshot, this appears not to be the right destination.

Reply:
Unless this is not the same for a pre-built vendored framework? How do we verify it works do you just upload a sample app to the app store?

Reply:
Yes, I had uploaded a sample app to the TestFlight.
And I test the crash it is correct symbolic.